packages/vite/src/node/plugins/esbuild.ts

```diff
@@ -27,6 +29,55 @@ export type ESBuildTransformResult = Omit<TransformResult, 'map'> & {
   map: SourceMap
 }
 
+type TSConfigJSON = {
```

Comment:
**question:** Doesn't have tsconfig its own interface?

Reply:
No😅
The `tsconfig` package uses the `any` type for the parsed JSON.

The `typescript` package does have an exported type for `compilerOptions`, but no exact match for the JSON file schema. And it's not a dependency of the `vite` package. So I had to define my own interface.